### ISSUE TITLE
Add support for team

### ISSUE THREAD

**Issue by j-martin** [shortcut-cli/shortcut-cli#287]

**Comment by omairvaiyani** [comment]
I have an open PR for this: https://github.com/shortcut-cli/shortcut-cli/pull/306

**Comment by panozzaj** [comment]
Looks like that PR has been merged, so this can likely be closed.

**Comment by charpeni** [comment]
Absolutely, let’s close this issue since the associated pull request is merged.
